### PR TITLE
fix: fixed volumes issue

### PR DESCRIPTION
**Pull request by 83wid**

### CODE REVIEW

**Review by Ysrbolles — APPROVED** [review]

**Comment by alizaynoune** [comment]
confirm mage

File: docker-compose.yml [hunk 1]
```diff
@@ -43,9 +45,4 @@ services:
...
       
 volumes:
-  db:
-    driver: local
-    driver_opts:
-      type: 'none' 
-      o: bind
-      device: $PWD/database/data
\ No newline at end of file
+  db:
```
Comment (alizaynoune) [inline docker-compose.yml:48]: add volumes:db

File: server/Dockerfile [hunk 2]
```diff
@@ -18,4 +18,6 @@ RUN echo "prisma db pull && prisma generate && yarn && yarn start:dev" > /start.
 
 WORKDIR /app
 
+RUN mkdir -p data
```
Comment (alizaynoune) [inline server/Dockerfile:21]: create foldel data

**Comment by alizaynoune** [comment]
done

**Pull request merged**